ui/admin/app/components/credential-stores/credential-store/nav/index.hbs

```diff
@@ -2,12 +2,19 @@
   <nav.link @route='scopes.scope.credential-stores.credential-store.index'>
     {{t 'titles.details'}}
   </nav.link>
-  {{#if (can 'navigate model' @model collection='credential-libraries')}}
-
+  {{#if @model.isVault}}
+    {{#if (can 'navigate model' @model collection='credential-libraries')}}
+      <nav.link
+        @route='scopes.scope.credential-stores.credential-store.credential-libraries'
+      >
+        {{t 'resources.credential-library.title_plural'}}
+      </nav.link>
+    {{/if}}
+  {{else}}
     <nav.link
-      @route='scopes.scope.credential-stores.credential-store.credential-libraries'
+      @route='scopes.scope.credential-stores.credential-store.credentials'
     >
-      {{t 'resources.credential-library.title_plural'}}
+      {{t 'resources.credential.title_plural'}}
```

Comment:
Should this tab be wrapped in a feature flag? Also, should we show a tab for static credential stores while the feature is not enabled?

Reply:
I don't think this requires feature-flag wrapper as we make the `static` non-clickable when the flag is disabled.  

<img width="1378" alt="Screen Shot 2022-07-20 at 2 54 11 PM" src="https://user-images.githubusercontent.com/15043878/180089093-c885a07a-13f1-42d4-a121-5fc2529cef19.png">

Reply:
regarding your second question - do you mean the `credentials` tab?

Reply:
Yes that is what I meant. However, with your previous comment about static credential stores not being clickable right now. I see no need for a change. Apologies for the confusion.